src/app/main-menu/main-menu.component.ts

```diff
@@ -90,9 +91,14 @@ export class MainMenuComponent implements OnInit, OnDestroy {
   }
 
   private openGlobalDialogInfo() {
-    // TODO openGlobalDialogInfo
-    console.log('openGlobalDialogInfo');
     this.itemClicked.emit('globalInfo');
+    const modalRef = this.modalService.open(ModalComponent, { id: 'bibliography' });
+    const modalComp = modalRef.componentInstance as ModalComponent;
+    modalComp.fixedHeight = true;
+    modalComp.modalId = 'bibliography';
+    modalComp.title = 'Bibliography';
+    modalComp.bodyContentClass = 'p-3';
+    modalComp.bodyComponent = BibliographyComponent;
```

Comment:
This kind of initialization should not be needed

Reply:
It's not that bad. I think it gives a lot of support in design.

Reply:
This is ok, since it is strictly linked to the properties of the `ModalComponent`, that was implemented in order to be re-used with the same basic UI, in many different ways. 
In my opinion, any improvements/changes of `ModalComponent` should be considered and developed in a different branch.

Reply:
IMHO not inside `openGlobalDialogInfo `

Reply:
Probably. But since the same method is used for opening other modals, I delegate this kind of refactoring to a separate task/branch